pages/index.tsx

```diff
@@ -43,8 +46,12 @@ const Index: NextPage<Props> = (props) => {
 
 Index.getInitialProps = async ({ req }) => {
   try {
+    const cookies = new Cookies(req?.headers.cookie);
```

Comment:
What's the benefit of using `universal-cookie` here instead of `req?.cookies[AUTH_COOKIE_NAME]`?

Doing a quick search, I don't see `universal-cookie` being used anywhere else... could we possibly remove it as a dependency entirely? @timmyichen

Reply:
Needs to work on the client too!

Reply:
Fair, I suppose until the CookieStore API becomes stable and common in most browsers, a package is probably the easiest solution. 👍🏼